File: Python/Strings/2866 - Criptotexto.py
"""
Autor: Pierre Vieira
Data da submissão: 30/01/2020 17:46:57
"""
for c in range(int(input())):
    string = input()
    s = ''
    for char in reversed(string):
        if char.islower():
            s += char
    print(s)

File: Kotlin/Strings/2866 - Criptotexto.kt
/*
* Autor: Pierre Vieira
* Data de submissão: 18/03/2023 16:44:58
*/

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        solution(readLine()!!)
        println()
    }
}

private fun solution(input: String) {
    for (i in (input.length - 1) downTo 0) {
        val currentChar = input[i]
        if (currentChar.isLowerCase()) print(currentChar)
    }
}